vendor/src/github.com/docker/notary/pkg/passphrase/passphrase.go

```diff
@@ -54,14 +71,20 @@ func PromptRetrieverWithInOut(in io.Reader, out io.Writer) Retriever {
 			fmt.Fprintln(out, tufRootKeyGenerationWarning)
 		}
 		if numAttempts > 0 {
-			if createNew {
-				fmt.Fprintln(out, "Passphrases do not match. Please retry.")
-
-			} else {
+			if !createNew {
 				fmt.Fprintln(out, "Passphrase incorrect. Please retry.")
 			}
 		}
 
+		// Figure out if we should display a different string for this alias
```

Comment:
UX change, allow the ability of changing what docker presents as the names for certain keys. Root is now offline key, and snapshot/target are tagging. These might change, we're currently in talks with Mary to make sure what is the simplest name for our end users.